WebRoot/skins/_base/base4/skin.properties

```diff
@@ -2020,3 +2020,17 @@ ChooserListViewCustomStyleColumn        = @FixBoxModel@; border:0; @TransparentB
 ChooserListViewCustomStyleHeader        = @ChooserListViewCustomStyleColumn@;
 ChooserListViewCustomStyleHeaderLabel   = @FontSize-slightly-bigger@; @Text@; text-transform:none; line-height:1; padding: 0 0 @SkinWrapperPadding@ 0;
 ChooserListViewCustomStyleContainer     = @PanelBorder@; @FixBoxModel@;
+
+#######################################################################
+#   Task List view
+#######################################################################
+TaskListViewAddNewTaskIcon               = display:inline-block; margin-right: 12px;
+TaskListViewNewTaskBannerWrapper         = @Text-hint@ @InactiveCursor@ @FontSize-big@ padding:16px 0 16px 0; margin-left:6px;
```

Comment:
I donot think using padding-left or margin-left will have any issues, here.
I tried using padding-left but effect is same with margin-left.
It depends on developer what he prefers.
If the white background is what you have in mind,  even the current behaviour in this situation is to having some white space on the left.